.github/workflows/mondayInformationals.yml

```diff
@@ -0,0 +1,33 @@
+name: "Monday Informationals"
+on:
+  schedule:
+    - cron: "0 14 * * MON"
+
+jobs:
+  monday_informationals:
+    name: Monday Informationals
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Dependencies
+        run: yarn global add --silent @artsy/cli@alpha
```

Comment:
Don't think it makes a big difference, but could we [use `npx` instead](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b) (which ships with node) and skip the install step?

Reply:
Ahh, that's a good idea. What if we were invoking the CLI multiple times within the same workflow? Would using npx install it N times?

Reply:
Yup, it would, so this would only be useful for one-off commands

Reply:
Would be cool to generate a docker image with the cli already installed so this is moot ya know?